lib/pgBackRest/Manifest.pm

```diff
@@ -822,7 +823,9 @@ sub build
         # lead to an invalid diff/incr backup later when using timestamps to determine which files have changed.  Offline backups do
         # not wait because it makes testing much faster and Postgres should not be running (if it is the backup will not be
         # consistent anyway and the one-second resolution problem is the least of our worries).
-        my $lTimeBegin = $oFile->wait(PATH_DB_ABSOLUTE, $bOnline);
+        my $lTimeBegin =
+            $oStorageDbMaster->can('protocol') ?
+                $oStorageDbMaster->protocol()->cmdExecute(OP_WAIT, [$bOnline]) : waitRemainder($bOnline);
```

Comment:
This is a perl function which asks if the named function exists.

Not sure I'm happy about this way of doing it (because of the string) but it's OK for now.